src/files/serializers.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+from .models import File
+from rest_framework import exceptions
+from django.conf import settings
+from .services import FileService
+
+class FileSerializer(serializers.ModelSerializer):
+    file = serializers.FileField(required=False)
+    class Meta:
+        model = File
+        fields = ['id', 'name', 'link', 'file']
+    def validate(self, attrs):
+        attrs = super().validate(attrs)
+        file = attrs.pop('file')
+        if file:
+            file_type = file.content_type.split('/')[1]
+            if file_type in settings.FILE_FORMAT:
+                if file.size > settings.UPLOAD_FILE_MAX_SIZE:
+                    raise exceptions.ValidationError('File type is not supported')
+            else:
+                raise exceptions.ValidationError('File type is not supported')
```

Comment:
lặp `exceptions.ValidationError('File type is not supported')`

Reply:
Dạ nếu else là not in thì vẫn là ko supported ạ